openedx/stanford/djangoapps/register_cme/tests/test_form.py

```diff
@@ -151,3 +151,31 @@ def test_export_countries(self, country):
         form = ExtraInfoForm(data)
         is_valid = form.is_valid()
         self.assertFalse(is_valid)
+
+    @data(None, 'ab', 'longsunet')
+    def test_affiliation_sunet_id(self, sunet_id):
+        data = self.data
+        data['affiliation'] = 'Stanford University'
+        data['stanford_department'] = 'Medicine'
+        data['sunet_id'] = sunet_id
+        form = ExtraInfoForm(data)
+        is_valid = form.is_valid()
+        self.assertFalse(is_valid)
+
+    @data(None, 'invalid_department')
+    def test_affiliation_stanford_department(self, stanford_department):
+        data = self.data
+        data['affiliation'] = 'Stanford University'
+        data['stanford_department'] = stanford_department
+        data['sunet_id'] = 'sunet_id'
+        form = ExtraInfoForm(data)
+        is_valid = form.is_valid()
+        self.assertFalse(is_valid)
+
+    @data('1')
```

Comment:
Hrm, we probably shouldn't use DDT if we aren't providing more than one value.

We could either:
- remove the `@data` decorator and just hard code the postal_code to the value
- add an additional value and also test for too long of an entry (I forget what the max value is)
  - if we do that, we'll want to modify the method name.